hax/hax/motr/hax.c

```diff
@@ -293,7 +293,10 @@ int m0_ha_pver_status(unsigned long long ctx,
 		FID_P(&pver_info.cpi_fid), pver_info.cpi_state);
 	enum m0_conf_pver_state status  = pver_info.cpi_state;*/
 	PyGILState_Release(gstate);
-	return 2;
+	// return status;
```

Comment:
Any specific reason we are not returning the real status?

Reply:
> @Shreya-18 @mssawant,
The patch to fetch data from btree and return via API is in this PR: https://github.com/Seagate/cortx-motr/pull/1499
which is currently in review. The commented-out code can be reverted back to normal when this PR merges to our feature branch 'dg_bytecount'.

That's why I had commented. 
Now that the motr's PR is merged, uncommented the code and sent the actual status.